client/src/features/results/Navigation.tsx

```diff
@@ -121,56 +121,70 @@ const Navigation: React.FC<INavigation> = ({
   }
 
   return (
-    <Row className="mx-0">
+    <Row className="mx-1 mt-3">
       <Col xs={12}>
-        <StyledNavbar data-testid="results-page-navbar">
-          <ul>
+        <StyledNavbar data-testid="results-page-navbar pb-0">
+          <Nav className="w-100 h-100 justify-content-between flex-row">
             {/* iterating over searchScopes to ensure the order the buttons are rendered */}
-            {Object.keys(searchScope).map((key) => (
-              <StyledNavLi
+            {Object.entries(searchScope).map(([key, value], ind) => (
```

Comment:
I'm getting an eslint error:
'ind' is defined but never used

Reply:
@gigamorph Fixed